SwiftBasicsExercise.playground/Pages/WorkingWithText_4.1.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,56 @@
+//MARK: Working with Strings & Characters
+
+import Foundation
+var inputString: String = "Simform"
+let inputString1 = "Swift", inputString2 = "example"
+var string1 = "Simform solutions"
+var messageInput = "Input : "
+var messageOutput = "Output : "
+
+//String & character exercise 1
+print("\n1 -->")
+print("Input : ", inputString)
+print(inputString.count < 2 ? messageInput + inputString : messageOutput + inputString.prefix(2))
+
+//String & character exercise 2
+print("\n2 -->")
+print("Input : ", inputString)
+let start = inputString.index(inputString.startIndex, offsetBy: 1)
+let end = inputString.index(inputString.endIndex, offsetBy: -1)
+let range = inputString[start..<end]
+print(inputString.count >= 2 ? messageOutput + inputString[start..<end] : "\(messageOutput) Length of string must be 2")
+
+//String & character exercise 3
+print("\n3 -->")
+print("Input : inputString1 = \(inputString1), inputString2 = \(inputString2)")
+
+let start_s1 = inputString1.index(inputString1.startIndex, offsetBy: 1)
+let start_s2 = inputString2.index(inputString2.startIndex, offsetBy: 1)
+let merged = inputString1[start_s1..<inputString1.endIndex] + inputString2[start_s2..<inputString2.endIndex]
```

Comment:
good that you have used range on the string, but we can just drop first and concatenate